samples/SampleApp/SampleApp.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
-  <Import Project="..\..\build\common.props" />
+  <Import Project="..\..\build\dependencies.props" />
```

Comment:
@natemcmaster pointed out we don't normally import `common.props` here, only `dependencies.props`. This was causing issues with user secrets because `common.props` has `GenerateUserSecretsAttribute` set to false.